model/vid_recognition.py

```diff
@@ -1,29 +1,21 @@
-import os.path
-import sys
-sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
-import face_recognition
+import face_recognition  # temporarily used in place of skull_detection
 import argparse
 import imutils
-import pickle
-import time
 import cv2
-from model import frame_recognition as fr
 from logger.base_logger import logger
-from utils.csv_logger import csv_logger
 
-# Description: Facial Recognition with video stream input
+# Description: Skull Recognition with video stream input
```

Comment:
We can use vid_recognition.py for phase 1, when detecting the skull for each frame in the video.